examples/templates/slack-template.twig

```diff
@@ -0,0 +1,25 @@
+{% set firstTicket = tickets | first %}
+{% set assignee = firstTicket.assignee %}
+
+{% if assignee.key %}
+    {% set salutation = "Hey, " ~ assignee.displayName ~ "(" ~ assignee.name ~ ")" %}
+{% else %}
+    {% set salutation = 'Hey Team' %}
+{% endif %}
+
+<header> {{ salutation }}, please have a look</header>
```

Comment:
Slack doesn't need (in fact you should not) use HTML tags on its messages.

Please, remove all HTML tags, and write each ticket message as a plain text; look at how it was in the previous src/ScrumMaster/Channel/Slack/MessageGenerator.php

Reply:
You are right :open_mouth: